plugins/processors/lotus_peer_id/peerid.go

```diff
@@ -0,0 +1,163 @@
+package lotus_peer_id
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs/lotus/rpc"
+	"github.com/influxdata/telegraf/plugins/processors"
+
+	"github.com/filecoin-project/lotus/api"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+const (
+	DefaultConfig_DataPath      = "${HOME}/.lotus"
+	DefaultConfig_APIListenAddr = "/ip4/0.0.0.0/tcp/1234"
+)
+
+var sampleConfig = fmt.Sprintf(`
+	## Provide the multiaddr that the lotus API is listening on. If API multiaddr is empty,
+	## telegraf will check lotus_datapath for values. Default: "%[1]s"
+	# lotus_api_multiaddr = "%[1]s"
+
+	## Provide the token that telegraf can use to access the lotus API. The token only requires
+	## "read" privileges. This is useful for restricting metrics to a seperate token.
+	# lotus_api_token = ""
+
+	## Provide the Lotus working path. The input plugin will scan the path contents
+	## for the correct listening address/port and API token. Note: Path must be readable
+	## by the telegraf process otherwise, the value should be manually set in api_listen_addr
+	## and api_token. This value is ignored if lotus_api_* values are populated.
+	## Default: "%[2]s"
+	lotus_datapath = "%[2]s"
+
+`, DefaultConfig_DataPath, DefaultConfig_APIListenAddr)
+
+type Exclude struct {
+	Measurement string `toml:"measurement"`
+}
+
+type LotusPeerIDTagger struct {
+	Config_DataPath      string `toml:"lotus_datapath"`
+	Config_APIListenAddr string `toml:"lotus_api_multiaddr"`
+	Config_APIToken      string `toml:"lotus_api_token"`
+
+	apiBackOffThrottle time.Duration
+	curAPIBackOff time.Time
+
+	initializedDefaults bool
+
+	Excludes []Exclude `toml:"exclude"`
```

Comment:
I have a gut feeling we'll want to add the PeerID tag to _most_ metrics passing through here so I am using a blacklist instead of a whitelist as the issue initially suggested.